docs/Users_Guide/reformat_stat_data.rst

```diff
@@ -46,160 +53,634 @@ Plots that are generated from the command line require a YAML configuration file
+
+
+     .. dropdown::  RHIST
+
+         **Ranked Histogram Counts**
+
+         From MET tools:
+            - ensemble-stat
+
+        Used by the following METplotpy plot types:
+            - rank histogram (rhist)
+
+     .. dropdown::  TCDIAG (with TCMPR)
+
+         **Tropcial Cyclone Diagnostics**
+         **Tropical Cyclone Matched Pairs**
+
+         From MET tool:
+            - tc-pairs
+
+        Used by the following METplotpy plot types:
+            - TCMPR (Tropical Cyclone Matched Pair) plots:
+
+            - mean line plot
+            - median line plot
+            - boxplot
+            - relative performance plot
+            - rank plot
+            - mean skill line plot
+            - median skill line plot
+
+     .. dropdown:: MPR
+
+         **Matched Pair data**
+
+         From MET tools:
+            - point-stat
+            - grid-stat
+
+        Used by the following METplotpy plot types:
+            - line
+            - scatter
+
+     .. dropdown:: DMAP
+
+         **Distance Map Statistics**
+
+         From MET tools:
+            - grid-stat
+
+        Used by the following METplotpy plot types:
+            - line
+            - contour
+            - scatter
+
+
 Description of Formats:
 -----------------------
 
-The reformatted data contains additional columns, based on the type of
-plot and line type.
+The reformatted data contains new columns, based on the type of
+plot and line type.  Formats fall into the following categories:
+
+    .. dropdown::  by statistic name, statistic value and confidence values
+
+       - Needed by the following plot types in METplotpy:
+
+         - line
+         - bar
+         - box
+         - contour
+         - performance diagram
+         - revision box
+         - revision series
+         - Taylor diagram
+
+       - Unlabelled columns in the original .stat file are placed under statistic name, statistic value,
+         and confidence level columns (in addition to the common stat columns):
+
+         - stat_name
+         - stat_value
+         - stat_ncl
+
+            - lower level normal confidence limit
+
+         - stat_ncu
+
+            - upper level normal confidence limit
+
+         - stat_bcl
+
+            - lower level bootstrap confidence limit
+
+         - stat_bcu
+
+            - upper level bootstrap confidence limit
+
+       - The original format is converted from wide format (where each row corresponds to a unique
+         set of common stat column values) to longer format (where there are multiple rows
+         corresponding to the same common stat column values).
+
+           - The common stat columns are common to all line types for point-stat, grid-stat, and ensemble-stat (model, vx mask, description, fcst lead, etc.)
+             as described in the MET User's Guide:
+
+              - table 11.1 (point-stat)
+              - table 12.1 (grid-stat)
+              - table 13.1 (ensemble-stat)
```

Comment:
```suggestion
              - `Table 13.1 <https://met.readthedocs.io/en/main_v12.0/Users_Guide/ensemble-stat.html#id1>`_ (ensemble-stat)
```

Reply:
modified and also used the camel-case version of the tool name, separated by '-'